src/mapping/mappers/Partners.js

```diff
@@ -0,0 +1,34 @@
+const { AggregateMapper } = require('../mapper');
+const { applyProfile, mcodeUtils09 } = require('../../utils');
+
+const resourceMapping = {
+    filter: () => true,
+    default: (resource, _context) => applyProfile(resource, mcodeUtils09.defaultProfile(resource.resourceType)),
+    mappers: [
+        {
+            filter: 'AllergyIntolerance',
+            exec: (resource, context) => {
+                if (!resource.substance.coding) {
+                    const text = resource.substance.text || '';
+                    resource.substance.coding = [{
+                        code: text,
+                        system: 'http://snomed.info/sct',
```

Comment:
Are we certain that what is coming over from partners is always a snomed code just in the text field?

Reply:
The `AllergyIntolerance`s from the environment did have SNOMED, although I don't suppose there is a way to know if this is a guarantee or not?

Not sure what to do in that case. Any thoughts?